### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by TahenyBELGUITH**
in this branch 
1️⃣ Added the contact form
2️⃣ Made sure that:
            1.           Name field accepts up to 30 chars.
            2.           Email field accepts only emails.
            3.           Message field accepts up to 500 chars.
            4.           All fields have placeholder text.
            5.           All fields have the correct type.
            6.           A user cannot submit the form if one or more of the fields is empty.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -146,11 +150,106 @@ <h3>project name goes here</h3>
...
+
+
+  <!-- contact form -->
+  <section class="form-section">
+   <div class="form-container">
+    <h2>Get started</h2>
+    <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+     If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+    <form action="https://formspree.io/f/xlezbalb" method="post">
+     <input type="text" name="name" placeholder="Your name" required>
```
Comment (nicupop729) [inline index.html:216]: - [ ]  As it is a project requirement, please set an attribute to the name input so the length could not be bigger than 30 characters long. :+1:

File: index.html [hunk 2]
```diff
@@ -146,11 +150,106 @@ <h3>project name goes here</h3>
...
+  </section>
+
+
+  <!-- contact form -->
+  <section class="form-section">
+   <div class="form-container">
+    <h2>Get started</h2>
+    <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+     If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+    <form action="https://formspree.io/f/xlezbalb" method="post">
```
Comment (nicupop729) [inline index.html:215]: The project requirements state as mandatory that to layout the elements on the page, we should use Flexbox. 

- [ ] In order to respect the mentioned requirements, please update your code to reflect a layout styled with Flexbox. :+1:

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️

Hi @TahenyBELGUITH, 

Good job so far!
You should be very proud of yourself, you have a very nice project. There are some issues that you still need to work on to go to the next project but you are almost there!



### Some highlights for you
* Correct GitHub flow practice ✔️ 
* Good commit messages ✔️ 
* Good description of the PR :heavy_check_mark: 
* Professional README ✔️ 
* Clean codebase ✔️ 

### Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## Hi @TahenyBELGUITH 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**